app/controllers/api/v2/bikes_search.rb

```diff
@@ -44,7 +44,6 @@ def set_proximity
           use :search_bikes
         end
         get "/", root: "bikes", each_serializer: BikeV2Serializer do
-          { "declared_params" => declared(params, include_missing: false) }
```

Comment:
I think it was critical that `include_missing` was passed here - but since this didn't break any tests, I'm assuming that an update made this no longer relevant. Or perhaps, include missing wasn't actually necessary.

Reply:
Line 47 is a no-op (likewise L67, L79, L185) since the hash literal isn't returned — I got the impression these might have been the response payloads for their respective endpoints originally but then more was added...?

Reply:
Ok! Ignore